packages/nextjs/hooks/scaffold-eth/useScaffoldContract.ts

```diff
@@ -5,10 +5,10 @@ import { useDeployedContractInfo } from "~~/hooks/scaffold-eth";
 import { Contract, ContractName } from "~~/utils/scaffold-eth/contract";
 
 /**
- * Gets a deployed contract by contract name and returns a contract instance
- * @param config - The config settings
- * @param config.contractName - Deployed contract name
- * @param config.walletClient - An viem wallet client instance (optional)
+ * Gets an instance of contract for the given contract name. Optional walletClient can be passed for doing write transactions.
+ * @param config - The config settings for the hook
```

Comment:
there's no `config` param in hook 🤷‍♂️

Reply:
Ohh lol yeah !! , even I was wondering initially but it seems a standard for documenting object properties : 

![Screenshot 2023-12-18 at 11 39 29 AM](https://github.com/scaffold-eth/scaffold-eth-2/assets/80153681/c4fa447d-1993-45fd-8cfb-548eabbf05b6)
https://jsdoc.app/tags-param